Amazon_schema.txt

```diff
@@ -0,0 +1,67 @@
+create table Addresses(AddressId int primary key identity(1,1),
+StreetAddress VARCHAR(100),
+City VARCHAR(50),
+State VARCHAR(50),
+PostalCode VARCHAR(10),
+Country VARCHAR(50));
+
+
+
+
+create table Users(UserId int primary key identity(1,1),
+Name varchar(50),
+PhoneNumber nvarchar(15),
+Email varchar(254) unique,
+AddressId int not null,
```

Comment:
Which data comes first , user or address ?

Reply:
According to this ,address comes first